src/modules/pages/components/ProfileTemplate/ProfileTemplate.css

```diff
@@ -43,3 +44,32 @@
     background-color: transparent;
   }
 }
+
+@media screen and query700 {
+  .main {
+    flex-direction: column;
+  }
+
+  .sidebar {
+    justify-content: center;
+    padding: 40px 0;
+    height: auto;
+    width: 100%;
+  }
+
+  .sidebar > div > div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .mainContent {
+    padding: 40px 30px;
+  }
+
+  .mainContent div:nth-child(2) {
+    justify-content: space-between;
+    row-gap: 10px;
+  }
+}
```

Comment:
Good work Will! I've checked the profile screen in different screen sizes and in my opinion, the space in between the colonies is a bit too big, example:

iPhone XR 
<img width="294" alt="Screen Shot 2022-07-05 at 12 40 36 PM" src="https://user-images.githubusercontent.com/71563622/177300032-00b0a2c9-2d6f-41bd-9dbb-d5dcbe685ca0.png">
 

But since there is no Figma design I don't know how this screen is supposed to look like. Generally, from what I have seen in other examples the space on the edges needs to be bigger than the space in between, examples: 

<img width="521" alt="Screen Shot 2022-07-05 at 12 59 03 PM" src="https://user-images.githubusercontent.com/71563622/177304073-1da40b56-3837-4ced-b685-7c8c8ac3acc6.png">
<img width="301" alt="Screen Shot 2022-07-05 at 12 59 22 PM" src="https://user-images.githubusercontent.com/71563622/177304113-595dca5b-fe82-4ad9-932f-96542093c971.png">

Only one screen had only one column of colonies (Samgsung Galxy S8+) instead of two columns. 

Samsung Galaxy S8+
<img width="306" alt="Screen Shot 2022-07-05 at 12 37 02 PM" src="https://user-images.githubusercontent.com/71563622/177299224-81b7ca6b-294a-4d68-afe9-8763fa6bc507.png">

Other than those two things, everything else looks good to me. :)

Reply:
Good suggestion, I have made it `space-around` instead